middleware/isClient.js

```diff
@@ -0,0 +1,9 @@
+const useContext = require("./useContext");
+const validateToken = require("./validateToken");
+
+module.exports = async (req, res, next) => {
+  const user = await useContext(req, res);
+  req.user = user;
+
```

Comment:
Although what you are trying to achieve here with the middleware is correct, I suggest that next time do not implement it, as the isManager `middleware` would suffice for the use cases of this project.